integration-tests/types/config/node/core.go

```diff
@@ -207,32 +207,32 @@ func WithVRFv2EVMEstimator(addr string) NodeConfigOpt {
 	return func(c *chainlink.Config) {
 		c.EVM[0].KeySpecific = evmcfg.KeySpecificConfig{
 			{
-				Key: ptr(ethkey.EIP55Address(addr)),
+				Key: Ptr(ethkey.EIP55Address(addr)),
 				GasEstimator: evmcfg.KeySpecificGasEstimator{
 					PriceMax: est,
 				},
 			},
 		}
 		c.EVM[0].Chain.GasEstimator = evmcfg.GasEstimator{
-			LimitDefault: ptr[uint32](3500000),
+			LimitDefault: Ptr[uint32](3500000),
 		}
 		c.EVM[0].Chain.Transactions = evmcfg.Transactions{
-			MaxQueued: ptr[uint32](10000),
+			MaxQueued: Ptr[uint32](10000),
 		}
 	}
 }
 
-func ptr[T any](t T) *T { return &t }
+func Ptr[T any](t T) *T { return &t }
```

Comment:
This might be better off in utils if we are making it public. I know I will need it in the automation test conversion.